Docs/source/running_cpp/parameters.rst

```diff
@@ -288,6 +288,12 @@ Particle initialization
     The mass of one `physical` particle of this species.
     If ``species_type`` is specified, the mass will be set to the physical value and ``mass`` is optional.
 
+* ``<species_name>.xmin,ymin,zmin`` (`float`) optional (default `geometry.prob_lo[0],[1],[2]`)
```

Comment:
I think the default is not `prob_lo/hi` as we saw in the implementation - but essentially `-inf/inf`:
https://github.com/ECP-WarpX/WarpX/blob/787bf0631809f6929bcd962cb8954063141e6490/Source/Initialization/PlasmaInjector.cpp#L82-L89

cc @atmyers

Reply:
Which gives the error when using the periodic boundary conditions.

Reply:
But I fixed the min, max to the box boundaries in this PR, that is why I updated the Docs with that information.

Reply:
ha, scrolling helps. sorry for that!